napalm_logs/transport/kafka.py

```diff
@@ -28,9 +27,11 @@ class KafkaTransport(TransportBase):
     '''
     Kafka transport class.
     '''
-    def __init__(self, addr, port):
-        self.bootstrap_servers = '{addr}:{port}'.format(addr=addr, port=port)
-        self.topic = KAFKA_PUBLISHER_TOPIC
+    def __init__(self, address, port, **kwargs):
+        for key, value in kwargs.items():
```

Comment:
Same comment as above, although not to be tackled here:

We should have an explicit topic -- if not provided by the user in the opts, we should use the default. Depending on the usecase of the transport, we can set this inside the base class, to avoid any complexity.

Also: let's not set everything from the kwargs, but only what we expect to be. Is there anything else other than the topic?